napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -502,7 +502,7 @@ def test_leaks_labels(qtbot, make_napari_viewer):
     )
     dr = weakref.ref(lr().data)
     viewer.layers.clear()
-    qtbot.wait(200)  # TODO: figure out exactly what we're waiting for
+    qtbot.wait(50)  # process events
```

Comment:
why reduce the threshold? curious

Reply:
I just changed this to `qapp.processEvents()`.   I didn't add that `wait` in the first place, but I did add the TODO recently.  There's really nothing asynchronous going on here. we just need to tell qt to cleanup dead objects.  `qtbot.wait()` enters a `while True: qapp.ProcessEvents(); sleep()` kind of loop... so I think a single `wait()` would have worked.  but `qapp.processEvents()` is clearer

Reply:
got it. thanks